front/src/app/components/forms/convention/ConventionForm.tsx

```diff
@@ -383,6 +433,12 @@ export const ConventionForm = ({
               data-matomo-name={domElementIds.conventionImmersionRoute.form({
                 mode,
               })}
+              onSubmit={(e) =>
+                handleSubmit(onSubmit, (errors) => {
+                  validateSteps("doNotClear");
+                  console.error(conventionValues, errors);
+                })(e)
+              }
```

Comment:
Tu doit pouvoir virer un niveau de fonction :
```suggestion
              onSubmit={handleSubmit(onSubmit, (errors) => {
                  validateSteps("doNotClear");
                  console.error(conventionValues, errors);
                })
              }
```